src/app/components/Tag/Tag.module.css

```diff
@@ -0,0 +1,20 @@
+.tag {
+  border: none;
+  border-radius: 3em;
+  padding: 0.2em 1em;
+  text-transform: uppercase;
+}
+
+.active {
+  background: var(--color-primary);
+  color: var(--bg-primary);
+}
+
+.inactive {
+  background: var(--bg-secondary);
+  color: var(--text-primary);
+}
+
+.count {
+  padding-left: 0.5ch;
```

Comment:
`ch` is an interesting unit, however it seems that it is not as relative as it could be. `em` might be the better choice here
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Values_and_Units#distance_units